### DISCUSSION TITLE
Hi, everybody, noticed that ddd is missing in Laravel 9

### DISCUSSION THREAD

**Discussion by Faks** [spatie/laravel-ignition#57] [Ideas]
Hi, everybody, noticed that ddd is missing in Laravel 9, does there are any plans to restore it back like it was in laravel 8 ?
Post Scriptum
Oskars Germovs.

**Comment by freekmurze** [comment]
We decided to remove it as we think not too many people are using it.

I wouldn't mind a PR that re-adds `ddd`

**Reply by MagnusDot** [reply]
many of my students, loved using this feature to help them learn laravel

**Reply by freekmurze** [reply]
Let one of your students PR it back in 👍
